app/services/invoices/payments/gocardless_service.rb

```diff
@@ -56,7 +58,10 @@ def update_payment_status(provider_payment_id:, status:)
         payment.update!(status: status)
 
         invoice_payment_status = invoice_payment_status(status)
-        payment.invoice.update!(payment_status: invoice_payment_status)
+        payment.invoice.update!(
+          payment_status: invoice_payment_status,
+          ready_for_payment_processing: invoice_payment_status != 'succeeded'
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)